### PR TITLE
Login

### PR DESCRIPTION
**Pull request by aleciabenjamin**
#### [PT Story:](https://www.pivotaltracker.com/story/show/165005630)
As a user
In order to access the application and its features
I would like to be able to log in

#### Description
- Create login form component
- Write component and integration tests
- Add styling to login form component

#### Screenshots:
Screenshot 2019-04-05 at 13 30 01

### CODE REVIEW

File: src/components/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const LoginForm = (props) => {
+	return (
+		<div className="form_div">
+				<img src={thymeLogo} alt="logo" className="thyme_logo" />
+				<form>
+					<div>
+						<input id="username" type="name" placeholder="Username" onChange={props.inputChangeHandler} className="user_input"></input>
+					</div>
+					<div>
+						<input id="password" type="password" placeholder="Password" onChange={props.inputChangeHandler} className="user_input"></input>
```
Comment (Joberg95) [inline src/components/LoginForm.js:14]: the placeholders dont look centered.

File: src/style/style.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	border: 2px solid black;
+	border-radius: 17px;
+	padding-left: 235px;
+	padding-right: 236px;
+	background-color: #5cb695;
+	padding-top: 7px;
+	padding-bottom: 7px;
+	color : white;
+	margin-top: 20px
+}
```
Comment (Joberg95) [inline src/style/style.css]: Why are you using a new css file when we have index.css ?

File: src/components/LoginLogic.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import LoginForm from './LoginForm'
+import { authenticate } from '../Modules/Auth'
+
+class LoginLogic extends Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			username: '',
+			password: '',
+			errorMessage: ''
```
Comment (Joberg95) [inline src/components/LoginLogic.js]: You can refactor this into this ``` 	state = {
			username: '',
			password: '',
			errorMessage: ''
	}; ```

File: src/style/style.css (hunk 2 above)
Comment (yanqiuzhang) [reply src/style/style.css]: done

File: src/components/LoginLogic.js (hunk 3 above)
Comment (yanqiuzhang) [reply src/components/LoginLogic.js]: done

File: src/components/LoginForm.js (hunk 1 above)
Comment (yanqiuzhang) [reply src/components/LoginForm.js:14]: done

**Comment by NiciP** [comment]
Reviewed

**Pull request merged**